autoware_adapi_v1_msgs/vehicle/msg/Kinematic.msg

```diff
@@ -0,0 +1,4 @@
+autoware_adapi_v1_msgs/GeoPointStamped geo_position
+geometry_msgs/PoseWithCovarianceStamped pose_with_covariance
+geometry_msgs/TwistWithCovarianceStamped twist_with_covariance
+geometry_msgs/AccelWithCovarianceStamped accel_with_covariance
```

Comment:
`with_covariance` can be omitted like messages in `geometry_msgs`.

Reply:
thanks for the commend, I will change it

Reply:
sorry for the long wait
I fixed in 48a7263